packages/@okta/vuepress-site/docs/guides/quickstart/create-org/cli/create-org.md

```diff
@@ -0,0 +1,21 @@
+Creating an org is easy using the [Okta CLI](https://github.com/oktadeveloper/okta-cli):
+
+1. After installing the Okta CLI, run the command `okta register`.
+
+2. You are prompted to supply the following information:
+
+	 - First name
+	 - Last name
+	 - Email address
+	 - Company
+
+3. Okta creates the org. The CLI returns your org URL, also known as your Okta domain, and lets you know that a verification code is being sent to your email.
```

Comment:
Can we structure this is a way that doesn't sound like `org URL` and `Okta domain` are the same thing?